### PR TITLE
New MAUI Examples Added

### PR DESCRIPTION
**Pull request by arctouch-alejandrovarela**
MAUI examples for next features:

- Accessibility action
- Accessibility live region
- Accessibility modal
- Element focus
- Input content type
- Input errors
- Input keyboard type
- Input label
- Keyboard order
- Keyboard shortcuts
- Media audio description
- Media captions live
- Media transcript
- Screen dark mode
- Screen orientation
- Text bold
- Text localization
- Element identification
- Element position
- Input cancellation
- Input gestures
- Input instructions
- Input motion
- Input predictable
- Linting
- Media audio control
- Screen animations
- Screen contrast
- Screen flashes
- Screen headers
- Screen labels
- Screen reflow
- Screen search
- Screen skip
- Screen timing
- Screen title
- Text element
- Text spacing
- Text truncation

### CODE REVIEW

File: src/data/code-samples/en/text-bold/net-maui.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Bold text - .NET MAUI
 
+While MAUI doesn't offer a way to detect if the user has enabled bold fonts, you can create a method and consume the Native API to detect user preferences.
```
Comment (ykozlovFivePro) [inline src/data/code-samples/en/text-bold/net-maui.md:3]: Maybe rewrite for consistency:
`In MAUI, there is no way to detect if the user has enabled bold font, but you can create a method and consume the Native API to detect user preferences.`

File: src/data/code-samples/en/text-localization/net-maui.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Localization - .NET MAUI
 
+MAUI offers several ways to handle localization. You can achieve this using the native approach via app resources, but it is recommended to use the .NET approach via [`RESX` files](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/localization). Make sure you follow all the instructions in the documentation to ensure it works correctly. After adding the strings into your `RESX` files, you can simply use them as follows:
```
Comment (ykozlovFivePro) [inline src/data/code-samples/en/text-localization/net-maui.md:3]: Let's replace the first sentence with:
`In MAUI, there are several ways to handle localization.`

File: src/data/code-samples/en/text-spacing/net-maui.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
 # Text spacing - .NET MAUI
 
+MAUI offers a few properties to adjust text spacing:
```
Comment (ykozlovFivePro) [inline src/data/code-samples/en/text-spacing/net-maui.md:3]: Let's replace the first sentence with:
In MAUI, there are several properties to adjust text spacing.

**Review by ykozlovFivePro — CHANGES_REQUESTED** [review]
Good overall! Nice having MAUI code samples for Appt!

Added few more comments to improve overall consistency between code samples and
suggestions to format the long lines of code to avoid having the horizontal scroll.

**Review by JJdeGroot — APPROVED** [review]

**Pull request merged**